lib/config/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmer-config"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
`wasmer-config` will need manual attention.
We need to observe semver.
We might have made some breaking changes mandating a `0.2.0`, can you check?

Reply:
Actually, I think we might have made 0 changes